aws-lc-rs/src/cipher/aes.rs

```diff
@@ -24,6 +24,8 @@ pub(crate) fn encrypt_block_aes(aes_key: &AES_KEY, block: Block) -> Block {
             aes_key,
         );
 
+        crate::fips::set_fips_service_status_unapproved();
```

Comment:
Would using [AES_ecb_encrypt](https://github.com/aws/aws-lc/blob/aa90e509f2e940916fbe9fdd469a4c90c51824f6/crypto/fipsmodule/aes/mode_wrappers.c#L84) instead of `AES_encrypt` cause this to become FIPS approved?

Reply:
Ah good catch, I checked and AES_ecb_encrypt does set the the indicator, will update the code to use that function.